Autopilot/Relay/AutoRelay_v0_01.c

```diff
@@ -139,6 +152,18 @@ void Init(void)
     IFS0bits.IC1IF = 0;         // Clear IC7 Interrupt Status Flag
     IEC0bits.IC1IE = 1;         // Enable IC7 interrupt
 
+    T1CONbits.TON = 0;          // Disable Timer
+    T1CONbits.TCS = 0;          // Select internal instruction cycle clock
+    T1CONbits.TGATE = 0;        // Disable Gated Timer mode
+    T1CONbits.TCKPS = 0b10;     // Select 1:64 Prescaler
+    
+    PR1 = 0xFFFF;               //Load the Period Value
+    TMR1 = 0x00;                //Clear Timer 1 Interrupt flag
```

Comment:
This is the Timer1 register (where it holds the counter), not the interrupt flag.